libcasr/src/report.rs

```diff
@@ -774,3 +774,230 @@ pub fn cluster_reports(casreps: &[CrashReport]) -> Result<Vec<u32>> {
 
     cluster_stacktraces(&traces)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::constants::*;
+    use crate::init_ignored_frames;
+
+    #[test]
+    fn test_report_display() {
+        // TODO: Fill all empty fields
+        let report = CrashReport {
```

Comment:
You can call  `CrashReport::new()` [function](https://github.com/ispras/casr/blob/master/libcasr/src/report.rs#L239) first.

Reply:
Done